amida-tech/orange-api
# Updated mockScripts

If this PR fixes a bug, you _must_ add test cases representative of the bug.

#### What's this PR do?
- This PR updates the existing mockScripts to provide the required data for the reports on Orange Web

### What does this PR not do? 
- Still outstanding is creating dummy accounts on production and setting up a cron job for this script to be run on a regular basis

#### Related JIRA tickets:
[ORANGE-1041](https://jira.amida.com/browse/ORANGE-1041)

#### How should this be manually tested?
If you have a local instance of the services with public registration set to true. simply run the mockscript using the command `node reportsMockScript.js` from inside the directory. 

- This will give you two logins, use the clinician login on your local orange-web instance and check the reports of the user to see the generated data. 

#### Any background context you want to provide?

#### Screenshots (if appropriate):
